ADT-SupplyChainDemo/README.md

```diff
@@ -1,6 +1,9 @@
 # Azure Digital Twins - Contoso Apparel 
-Overview and Demonstration:
-[![Supply Chain Demo](./images/supply-chain.png)](https://www.youtube.com/watch?v=ScmK-bKJ4MIe)
+
+## Overview
+Azure Digital Twins is a platform as a service (PaaS) offering that enables the creation of knowledge graphs based on digital models of entire environments. These environments could be buildings, factories, farms, energy networks, railways, stadiums, and more—even entire cities. These digital models can be used to gain insights that drive better products, optimized operations, reduced costs, and breakthrough customer experiences. 
+
+The following demos provide a typical implementation via a standalone web app and also a developer focussed version for users to experiment with. 
```

Comment:
"The following demos provide implementation guidance via a standalone demo web app and a developer focused backend sample for users to experiment with."